pipeline/builder/types/types.go

```diff
@@ -57,15 +56,15 @@ type Stage interface {
 type ManifestStage struct {
 	StageMetadata
 
-	Account                 string            `json:"account"`
-	CloudProvider           string            `json:"cloudProvider"`
-	Location                string            `json:"location"`
-	ManifestArtifactAccount string            `json:"manifestArtifactAccount"`
-	ManifestName            string            `json:"manifestName"`
-	Manifests               []json.RawMessage `json:"manifests"`
-	Moniker                 Moniker           `json:"moniker"`
-	Relationships           Relationships     `json:"relationships"`
-	Source                  string            `json:"source"`
+	Account                 string           `json:"account"`
+	CloudProvider           string           `json:"cloudProvider"`
+	Location                string           `json:"location"`
+	ManifestArtifactAccount string           `json:"manifestArtifactAccount"`
+	ManifestName            string           `json:"manifestName"`
+	Manifests               []runtime.Object `json:"manifests"`
```

Comment:
how

Reply:
Magic.

Reply:
Since this just an interface that is fulfilled by the k8s object types (`appsv1.Deployment` for example), Go will use the struct tags on the underlying type when rendering the JSON. This is how the stages work in the builder/types package too.